k-distribution/tests/regression-new/io-ocaml/test.k

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) 2013-2019 K Team. All Rights Reserved.
```

Comment:
this should be a symlink pointing to the other file.

Reply:
The two files are different. For some reason, the ocaml-backend fails with this error:
```
File "realdef.ml", line 2125, characters 180-195:
Error: This expression has type Constants.K.kitem
       but an expression was expected of type
         Constants.k = Constants.K.kitem list
```
realdef.ml area: `isTrue (evalisKResult((varHOLE_184)) config (-1))` (`varHOLE_184` is the problematic term)
test.k area: `context assertEqual(HOLE, _:KItem)`
You are right about the tests. They can be symlinks, but I don't know where this is comming from, so I just created another test.

Reply:
Can you please post the diff between the two files then? That might help me figure out what is causing the error you're getting.

Reply:
I removed the context declaration and the 3 rules at the end which seemed inactive.
The error points to the `HOLE` from the context.
Or are you talking about the diff from the resulting realdef.ml files?

Reply:
Actually realdef.ml doesn't make sense because the other one is compiled with java.